library-core/src/main/java/com/mikepenz/aboutlibraries/entity/Library.kt

```diff
@@ -4,6 +4,7 @@ package com.mikepenz.aboutlibraries.entity
 data class Library(
         var definedName: String,
         var isInternal: Boolean = false,
+        var isPlugin: Boolean = false,
```

Comment:
In general, prefer immutable data types where possible. Immutable changes can be done via `copy(paramName = paramValue)`